### ISSUE TITLE
Error with `sphinx-notfound-page` 1.0.3 - 'tuple' object has no attribute 'default'

### ISSUE THREAD

**Issue by dgarcia360** [readthedocs/sphinx-notfound-page#240]
## Context

I get the following error in some Sphinx-based projects with the [latest release](https://github.com/readthedocs/sphinx-notfound-page/releases/tag/1.0.3) of the extension:

> Extension error (notfound.extension):
> Handler <function validate_configs at 0x7f69c2a4caf0> for event 'config-inited' threw an exception (exception: 'tuple' object has no attribute 'default')

More details, I'm using:

- Python 3.12
- Sphinx 7.2.6
- `notfound_urls_prefix` values in `conf.py` are set to `None` or ``.

## Debugging

The issue might be here: https://github.com/readthedocs/sphinx-notfound-page/commit/64ed88f43b4aa0299ec168a60ff5ecfd2d8e1bc7#diff-83bfdb6415a043b1888e7ce5240bd78ea1f236af2f7094fcb5e7702f8672aa32R274

Is trying to access `app.config.values.get("notfound_urls_prefix").default`, but I'm not sure if the default is being defined.

**Comment by humitos** [comment]
We talked about this issue on https://github.com/readthedocs/sphinx-notfound-page/pull/236#pullrequestreview-2073228098. I thought it was already fixed 🤔

**Comment by dgarcia360** [comment]
Thanks @humitos! I get the build working updating to [Sphinx 7.3.0](https://www.sphinx-doc.org/en/master/changes.html#release-7-3-0-released-apr-16-2024).

Most likely, this is the version where `_Opt` was included:

- 7.3.0: https://github.com/sphinx-doc/sphinx/blob/v7.3.0/sphinx/config.py#L101
- 7.2.6: https://github.com/sphinx-doc/sphinx/blob/v7.2.6/sphinx/config.py

I'm happy  to submit a PR changing:

```
if sphinx.version_info >= (7, 2)
```

To:

```
if sphinx.version_info >= (7, 3)
```

**Comment by dgarcia360** [comment]
PR with the fix: https://github.com/readthedocs/sphinx-notfound-page/pull/241

**Comment by humitos** [comment]
> `notfound_urls_prefix` values in `conf.py` are set to `None` or ``.

I'm not yet sure to understand why this wasn't catch by the test we have for this config: https://github.com/rtfd/sphinx-notfound-page/blob/ccdda4977159004aca945bdd1c249b08cf685afe/tests/test_urls.py#L162-L201

We will need to research a little more that so we can write a test case that pass/break depending on the code we are fixing.

**Comment by humitos** [comment]
I released [1.0.4](https://pypi.org/project/sphinx-notfound-page/1.0.4/), can you please confirm that it works as expected? 🙏🏼

**Comment by dgarcia360** [comment]
Works fine, thanks!
